addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/protocol/QueuedSend.java

```diff
@@ -134,7 +134,7 @@ public void run() {
                 Thread.sleep((item.custom_delay_time != 0) ? item.custom_delay_time : delay_between_commands);
             } catch (InterruptedException e) {
                 if (!willbeclosed) {
-                    logger.error("Queue sleep failed: " + e.getLocalizedMessage());
+                    logger.error("Queue sleep failed: {}", e.getLocalizedMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace